src/assets/scripts/client/navigationLibrary/NavigationLibrary.js

```diff
@@ -0,0 +1,93 @@
+import PositionModel from '../base/PositionModel';
+import FixCollection from './Fix/FixCollection';
+import StandardRouteCollection from './StandardRoute/StandardRouteCollection';
+
+/**
+ * @class NavigationLibrary
+ */
+export default class NavigationLibrary {
+    /**
+     * @constructor
+     * @for NavigationLibrary
+     * @param airportJson {object}
+     */
+    constructor(airportJson) {
+        /**
+         *
+         *
+         * @property _referencePosition
+         * @type {PositionModel}
+         * @default null
+         */
+        this._referencePosition = null
+
+        /**
+         *
+         *
+         * @property _sidCollection
+         * @type {StandardRoute}
+         * @default null
+         */
+        this._sidCollection = null;
+
+        /**
+         *
+         *
+         * @property _starCollection
+         * @type {StandardRoute}
+         * @default null
+         */
+        this._starCollection = null;
```

Comment:
I just wrote a whole blob about how fixes can exist by themselves, blah blah blah.  That's not what you're asking here.  You are asking about SIDs/STARs.

In a word, no.  You are correct, sids and stars _can_ belong to an airport and _can_ be airport specific.  But aren't there also times when a single STAR can be used for multiple airports within a close area?  I believe there are one or two like that here in minneapolis for kmsp and kstp (or even kfcm).

Either way, sids and stars are _related_ to an airport, but don't really need an airport to work.  sids and stars are really just a specific arrangement of fixes, right?  

That's why I love this `NavigationLibrary` class.  It can hold all of our fix related containers.  I still haven't completely diverted the river on this one yet, so this could change, but I like having all these concerns together for now.